src/non_markov_localization.cpp

```diff
@@ -1362,7 +1365,7 @@ void NonMarkovLocalization::TrimEpisode(const int pose_index) {
 
 
 void NonMarkovLocalization::SensorUpdate(
-    const PointCloudf& point_cloud, const NormalCloudf& normal_cloud) {
+    const PointCloudf& point_cloud, const NormalCloudf& normal_cloud, const ros::Time &laser_time) {
```

Comment:
The main EnML library is ROS-free, so we don't pass in ROS objects. You can use a `double` epoch timestamp instead.

Reply:
I opted to create a new structure with integer seconds and nanoseconds so as not to introduce floating point and also keep it compatible with ROS in case we want to convert the data back to ROS at some point.